pkg/controller/grafanadashboard/dashboard_controller.go

```diff
@@ -207,3 +255,36 @@ func (r *ReconcileGrafanaDashboard) updatePhase(cr *i8ly.GrafanaDashboard, phase
 	err := r.client.Update(context.TODO(), cr)
 	return reconcile.Result{}, err
 }
+
+func (r *ReconcileGrafanaDashboard) updateStatus(cr *i8ly.GrafanaDashboard, status i8ly.GrafanaDashboardStatusMessage) {
```

Comment:
There is already a function to append a status message here: https://github.com/integr8ly/grafana-operator/blob/master/pkg/controller/grafana/pluginsHelper.go#L59

maybe we can remove one of them and move the other into common and use it in both places?